source/reference/known-issues.txt

```diff
@@ -68,3 +68,23 @@ Given the exact same |sql| query, the |bi-short| does not return
       GROUP BY class;
       
       Empty set (0.00 sec)
+
+
+Mapping Potentially Null Values at Different Object Depths
```

Comment:
I'm thinking that this title maybe should describe the actual error (like the error above) since users are likely arriving at this page expecting to locate the issues that they're experiencing.

maybe something like "BI Connector Returns ``NULL`` Values due to Mapping Error"?